photon-core/src/test/java/org/photonvision/vision/target/TargetCalculationsTest.java

```diff
@@ -112,45 +120,79 @@ private static Stream<Arguments> testYawPitchCalcArgs() {
                 Arguments.of(0, 10),
                 Arguments.of(10, 10),
                 Arguments.of(-10, -10),
-                Arguments.of(30, 45),
-                Arguments.of(-45, -20));
+                // reduced magnitude to stay inside imager
+                Arguments.of(18, 21),
+                Arguments.of(-23, -16));
     }
 
-    private static double[] testCameraMatrix = {240, 0, 320, 0, 240, 320, 0, 0, 1};
-
     @ParameterizedTest
     @MethodSource("testYawPitchCalcArgs")
     public void testYawPitchCalc(double yawDeg, double pitchDeg) {
-        Mat testCameraMat = new Mat(3, 3, CvType.CV_64F);
-        testCameraMat.put(0, 0, testCameraMatrix);
+        JsonMatOfDouble testCameraMatrix =
+                new JsonMatOfDouble(
+                        3, 3, new double[] {1142.341323, 0, 621.384201309, 0, 1139.92214, 349.897631, 0, 0, 1});
+        JsonMatOfDouble testDistortion =
+                new JsonMatOfDouble(
+                        5,
+                        1,
+                        new double[] {
+                            0.186841202993646,
+                            -1.482894102216622,
+                            0.005692954661309707,
+                            0.0006757267756945662,
+                            2.8659664873321287
+                        });
+        double IMAGER_WIDTH = 1280, IMAGER_HEIGHT = 720;
+        var testCameraCal =
+                new CameraCalibrationCoefficients(
+                        imageSize,
+                        testCameraMatrix,
+                        testDistortion,
+                        new double[0],
+                        List.of(),
+                        new Size(),
+                        0,
+                        CameraLensModel.LENSMODEL_OPENCV);
+
         // Since we create this translation using the given yaw/pitch, we should see the same angles
         // calculated
         var targetTrl =
                 new Translation3d(1, new Rotation3d(0, Math.toRadians(pitchDeg), Math.toRadians(yawDeg)));
         // NWU to EDN
         var objectPoints =
-                new MatOfPoint3f(new Point3(-targetTrl.getY(), -targetTrl.getZ(), targetTrl.getX()));
+                new MatOfPoint3f(new Point3(targetTrl.getY(), targetTrl.getZ(), targetTrl.getX()));
         var imagePoints = new MatOfPoint2f();
```

Comment:
> - figure out why the unit test needs a negative removed

This assertion check might be using the wrong sign, the 3d object point is created with polar coordinate angles in WPILib NWU (in the image, pitch positive down and yaw positive left) but checked against the yaw/pitch calculated by photonvision (pitch positive up, yaw positive right).